TrafficSimulator/Assets/GraphChartFeed.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+using ChartAndGraph;
+using System.Collections;
+
+public class GraphChartFeed : MonoBehaviour
+{
+    void Start ()
+    {
+        GraphChartBase graph = GetComponent<GraphChartBase>();
+        if (graph != null)
+        {
+            
+            graph.Scrollable = false;
+            graph.HorizontalValueToStringMap[0.0] = "Zero"; // example of how to set custom axis strings
+            graph.DataSource.StartBatch();
+            graph.DataSource.ClearCategory("Player 1");
+            graph.DataSource.ClearAndMakeBezierCurve("Player 2");
+            
+            for (int i = 0; i <5; i++)
+            {
+                graph.DataSource.AddPointToCategory("Player 1",i*5,Random.value*10f + 20f);
```

Comment:
```suggestion
                graph.DataSource.AddPointToCategory("Player 1", i*5, Random.value * 10f + 20f);
```

Reply:
Leave this for now, will change when finished setting up everything in this window.